impexp-core/src/main/java/org/citydb/citygml/exporter/database/content/DBCityObject.java

```diff
@@ -472,10 +482,79 @@ private void postprocess() throws CityGMLExportException, SQLException {
 		generalizesToExporter.executeBatch();
 	}
 
+	private void delegateADEProperties(ObjectContext context, long objectId) throws SQLException, CityGMLExportException {
+		// we only have to query and delegate ADE properties here if the
+		// ADE feature is a direct child of AbstractCityObject and, thus,
```

Comment:
better to say here "direct child of AbstractCityObject or AbstractSite"

Reply:
good point, implemented with 1b812ecd32b23d66efdfd99e17026bc77d0a3f6a